code/starfield/starfield.cpp

```diff
@@ -1750,22 +1751,24 @@ void stars_draw_debris()
 
 	for (i=0; i<MAX_DEBRIS; i++, d++ ) {
 		if (!d->active)	{
-			d->pos.xyz.x = f2fl(myrand() - RAND_MAX_2);
-			d->pos.xyz.y = f2fl(myrand() - RAND_MAX_2);
-			d->pos.xyz.z = f2fl(myrand() - RAND_MAX_2);
+			d->pos.xyz.x = f2fl(Random::next() - Random::HALF_MAX_VALUE);
+			d->pos.xyz.y = f2fl(Random::next() - Random::HALF_MAX_VALUE);
+			d->pos.xyz.z = f2fl(Random::next() - Random::HALF_MAX_VALUE);
 
 			vm_vec_normalize(&d->pos);
 
 			vm_vec_scale(&d->pos, MAX_DIST);
 			vm_vec_add2(&d->pos, &Eye_position );
 			d->active = 1;
+			// DISCUSSME: is the following line correct? What was the author trying to do?
 			d->vclip = i % MAX_DEBRIS_VCLIPS;	//rand()
```

Comment:
I wonder if the author intended what would now be expressed as
```
d->vclip = Random::next(MAX_DEBRIS_VCLIPS);
```
but I take it I should leave this line alone, including leaving the existing comment as is?

Reply:
This is the last remaining `DISCUSSME` btw.

Reply:
I would say leave it as it is. We can change is later if needed but this is working now so no need to change anything around this.